nhais-adaptor/edifact/outgoing/models/death/message_death.py

```diff
@@ -0,0 +1,62 @@
+from edifact.helpers import date_formatter as date_formatter
+from edifact.outgoing.models.message import MessageSegmentRegistrationDetails, MessageSegmentPatientDetails, Message
+from edifact.outgoing.models.segment import Segment
+
+
+class MessageSegmentDeathPatientDetails(MessageSegmentPatientDetails):
+    """
+    A specialisation of the MessageSegmentPatientDetails class for the purpose of a Death
+    """
+
+    def __init__(self, id_number):
+        """
+        :param id_number: OPI official Payment Id (existing NHS Number)

```

Comment:
Payment ID?

Reply:
Haha I've never noticed that. It's something I've copied from the edifact spec. I'll have a look in there to see what it says.

Reply:
It's Patient in the spec. Don't know where payment came from 🤷‍♂